### PR TITLE
Vendor Payout SMS

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Send SMS when a vendor payout happens  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

File: app/server/locale/general_sms.en.yml [hunk 1]
```diff
@@ -4,3 +4,4 @@ en:
     grassroots: Hello %{first_name} you have been registered on Sarafu Network! Your balance is %{balance} %{token}. To use dial *384*96# Safaricom or *483*46# Airtel. For help 0757628885
   terms: By using the service, you agree to the terms and conditions at https://withsempo.com/legal/platform-terms.
   pin_reset: Dial *384*96# Safaricom or *483*46# Airtel to change your PIN
+  payout_message: Hello %{first_name}, a total of %{amount} %{token} has been deposited into your account.
```
Comment (michielderoos) [inline app/server/locale/general_sms.en.yml:7]: What should this copy be?

**Comment by michielderoos** [comment]
[SMS when withdrawal started and complete](https://trello.com/c/FYZLpCMi/928-sms-when-withdrawal-started-and-complete)

**Comment by michielderoos** [comment]
[SMS when withdrawal started and complete](https://trello.com/c/FYZLpCMi/928-sms-when-withdrawal-started-and-complete)

File: app/server/locale/general_sms.en.yml (hunk 1 above)
Comment (tristanhcole) [reply app/server/locale/general_sms.en.yml:7]: This seems reasonable

**Review by tristanhcole — APPROVED** [review]
LGTM!

**Pull request merged**